client/src/components/Table/table.css

```diff
@@ -0,0 +1,40 @@
+/* @import url("https://fonts.googleapis.com/css?family=Volkhov"); */
+body {
```

Comment:
I usually prefer to put body styling in a top-level `site.css` or `main.css`.  Then it's easy to find and it's always included (if you stopped importing Table, your app would suddenly look very different 😀).  That's usually where I put other 'global' styling, like 
```
* {
  box-sizing: border-box;
}
```
and things like that.

Reply:
@tomduggan85 Although I'd not prefer using the global reset (that's what I named it :) )
```css
* {
  ...
}
```
I've been watching this course about CSS, and it said very clearly that there should never be a CSS selector that is bad enough to make a noticeable performance drop-down, but the selector that takes the most time regarding loading the CSS selectors is actually the reset section.
Again, there should not (unless you are very determined to) ever be a CSS selector that is bad enough to affect your performance. I just thought it's worthy to speak of :) .

The alternative is calling every selector by name, like, we almost never use tags like `aside` and `article` ...etc, so instead of calling the global reset, we could say:
```css
h1, h2, h3, div, span, ul, ol, li {
```

Reply:
Ah yep, good point!  If I remember, `*` is the slowest selector, then tag (i.e. `li {`, then class, then id.

I was curious so I just looked at the reset.css that we've been using.  We're using this one: http://meyerweb.com/eric/tools/css/reset/ 
.. and it doesn't have any wildcard selectors (or box-sizing at all).